src/Routers/FunctionsRouter.js

```diff
@@ -18,21 +18,21 @@ function parseObject(obj) {
     return Object.assign(new Date(obj.iso), obj);
   } else if (obj && obj.__type == 'File') {
     return Parse.File.fromJSON(obj);
-  } else if (obj && obj.__type == 'Pointer') {
+  } else if (obj && obj.__type == 'Pointer' && config.encodeParseObjectInCloudFunction) {
```

Comment:
@mtrezza 

Not sure if i'm missing something, but i get this error

```
TypeError: Cannot read properties of undefined (reading 'encodeParseObjectInCloudFunction')
    at parseObject (/var/www/asdf/node_modules/parse-server/src/Routers/FunctionsRouter.js:21:55)
```

Haven't investigated yet.